### ISSUE TITLE
due to up coming QTS 5, please modify template qpkg.conf

### ISSUE THREAD

**Issue by nephaste** [qnap-dev/QDK#27]
Hi Ed

can you please modify qpkg.conf template

# Maximum QTS version requirement
QTS_MAX_VERSION="5.0.0"

to 

# Maximum QTS version requirement
#QTS_MAX_VERSION="5.0.0"

thanks a lot

**Comment by OneCDOnly** [comment]
Agree, this would be a good thing. QTS 5.0.0 is the current version. What happens when QTS 5.0.1 comes along? 😉

**Comment by edhongcy** [comment]
Hi @OneCDOnly 
I remove QTS max version requirement on v2.3.12
[e6256e7cacf985ab17dce22101d16369509038d6](https://github.com/qnap-dev/QDK/commit/e6256e7cacf985ab17dce22101d16369509038d6)
